src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -1293,7 +1294,7 @@ void MacroAssembler::lookup_interface_method_stub(Register recv_klass,
   // Finally, scan_temp contains holder_klass vtable offset
   bind(L_holder_found);
   ldrw(method_result, Address(scan_temp, ooffset - ioffset));
-  add(recv_klass, recv_klass, (itable_index << 3) + in_bytes(itableMethodEntry::method_offset()) - vtable_start_offset - ioffset);
+  add(recv_klass, recv_klass, (itable_index << LogBytesPerWord) + in_bytes(itableMethodEntry::method_offset()) - vtable_start_offset - ioffset);
```

Comment:
Maybe use `itable_index * wordSize` instead of `itable_index << LogBytesPerWord` to look the same as in the x86 variant of `lookup_interface_method_stub`. 
Also a comment that `- vtable_start_offset - ioffset` is used to restore the original value of `recv_klass` would be useful.